docs/src/components/ComponentPage/WebComponentPage.js

```diff
@@ -36,7 +36,7 @@ type Props = {
     blurb: Node,
     examples: Node,
   }>,
-  readme: string,
+  readme: ?string,
```

Comment:
The alignment page doesn't have a readme, so I had to make it optional for web components.